via/services/secure_link.py

```diff
@@ -31,6 +42,14 @@ def request_is_valid(self, request):
 
         return True
 
+    def sign_url(self, url):
+        """Get a signed URL (if URL signing is enabled)."""
+
+        if self._signed_urls_required:
+            return self._via_secure_url.create(url)
+
+        return url
```

Comment:
:+1: We should probably remove `signed_urls_required`. I'm guessing it's always on